dot/node_test.go

```diff
@@ -194,9 +198,16 @@ func TestInitNode_LoadGenesisData(t *testing.T) {
 	err := InitNode(cfg)
 	require.NoError(t, err)
 
+	ctrl := gomock.NewController(t)
+	telemetryMock := NewMockClient(ctrl)
+	telemetryMock.EXPECT().SendMessage(gomock.AssignableToTypeOf(&telemetry.NotifyFinalizedTM{}))
```

Comment:
If there is only one message being sent, you could easily (I think) expect the actual message instead of just its type